### ISSUE TITLE
Enable issues for the mobile app repo

### ISSUE THREAD

**Issue by 12people** [FilenCloudDienste/filen-desktop#126]
**Is your feature request related to a problem? Please describe.**
There doesn't seem to be a clear way to report issues or file feature requests with the mobile app.

**Describe the solution you'd like**
An issues page on the [mobile app Github repo page](https://github.com/FilenCloudDienste/filen-mobile).

**Describe alternatives you've considered**
There are potentially other ways to report bugs, but all the other Filen Github repos seem to have their own Issues page, so why shouldn't the mobile app have one?

**Comment by Dwynr** [comment]
We are reworking our mobile app. Issue tab will be re-enabled once the updated version is merged into the current repo.
